### ISSUE TITLE
自动展示功能会被自动取消

### ISSUE THREAD

**Issue by huanghonghaiaa88** [ankitects/anki#3354]
当鼠标点击软件窗口外，自动展示功能会被自动取消，希望不会被取消。
版本是 24.06.3。

**Comment by OuOu2021** [comment]
![image](https://github.com/user-attachments/assets/3bd83ff6-5f99-494f-acae-77ecfb42d830)
事实上代码就是这样写的，但我不知道为什么要这么设计

**Comment by dae** [comment]
The goal is to prevent users from accidentally having all their cards answered when they switch away to a different window / start editing the current card, etc.

**Comment by huanghonghaiaa88** [comment]
其实对于某部分用户来说，是比较希望能后台运行的。比如，我正在上班摸鱼，我想打开两个窗口，一个窗口编辑文档，一个窗口使用Anki学习（Anki不会被中断）。

**Comment by OuOu2021** [comment]
> The goal is to prevent users from accidentally having all their cards answered when they switch away to a different window / start editing the current card, etc.

Would it better to suspend auto_advance or pause the timer temporarily instead of disabling auto_advance when the focus is changed?

**Comment by dae** [comment]
That would result in auto advance resuming when the user switches back to the app, which I think some users wouldn't appreciate. It also doesn't help you, if your use case is to open Anki and then switch another app into the foreground.

**Comment by Thiago-Morais** [comment]
I agree that it is very inconvenient to have to enable `Auto Advance` every time I start studying a different deck or alt+tab (for searching more about a card or even looking at a dictionary)

It makes sense to ignore the `Question action`/`Answer action` when alt+tab for the single current card, but disabling auto advance completely, even when changing between decks, seems counterintuitive

**Comment by dentistfrankchen** [comment]
@OuOu2021 源代码是哪个文件？能否自己DIY一下把代码注释掉？谢谢大佬，这个很重要
